README.md

```diff
@@ -117,4 +117,15 @@ cd zlux/zlux-app-server/bin
 `explorer-jes` root already have sample `pluginDefinition.json` & will have `web` folder after `build`.
 
 
+## Enable Redux logs
+Either use Redux Tool Browser Extension in your browser or enable redux logs by setting `enableReduxLogger` variable `true` in your local storage.
+
+Run this in browser console
```

Comment:
Add the fact that you need to reload the page too

Reply:
but you can leave that set in your browser and you will always see that, we dont clean that up
If we introduce configuration section, we can use that too

It can be other enhancement, if we can add middleware after store is intialized.
Its Single Page app, so all the behavior configured when we load website

Reply:
ok adding comment to refresh browser

Reply:
updated instructions